### PR TITLE
Show chunks faster and reduce amount of invisible chunks (Issue #1886).

### PR DESCRIPTION
**Pull request by flo**
### Contains

Improves the situation described in issue #1886.  In single player chunks become now visible when the nearby chunks got loaded and the chunk in the current location got light merged.

In multiplayer the amount of needed chunks before the first chunk becomes visible should be reduced from 7x7x7 to 5x5x5. It is an improvement but there is still room for further optimization.

### How to test

Change the view distance to "Legally Bild (5x5x5)" and then start a single player game. Do this once with and without the changes of this PR. Without this PR only a single chunk is visible. With this PR 9 chunks are visible (3x3x3).

### CODE REVIEW

**Comment by GooeyHub** [comment]
Hooray Jenkins reported success with all tests good!

File: engine/src/main/java/org/terasology/rendering/world/RenderableWorldImpl.java [hunk 1]
```diff
@@ -377,7 +377,14 @@ public void dispose() {
     }
 
     private boolean isChunkValidForRender(RenderableChunk chunk) {
-        return chunk.isReady() && chunk.areAdjacentChunksReady();
+        if (!chunk.isReady()) {
+            return false;
+        }
+        boolean surroundingChunksLoaded = worldProvider.getWorldViewAround(chunk.getPosition()) == null;
```
Comment (vampcat) [inline engine/src/main/java/org/terasology/rendering/world/RenderableWorldImpl.java]: Doesn't this variable actually represent `surroundingChunksNotLoaded` ?

**Comment by GooeyHub** [comment]
Hooray Jenkins reported success with all tests good!

**Comment by Cervator** [comment]
Checked this out locally, can confirm legally blind now feels substantially less blindy :-)

Also threw in a quick commit to add `configs` as an excluded dir in IntelliJ, forgot to do that when merging another thing.

**Pull request merged**